### DISCUSSION TITLE
When will Godot support RISC-V SBC?

### DISCUSSION THREAD

**Discussion by DSOE1024** [godotengine/godot-proposals#8886] [Platforms]
Now many RISCV-V SBCs have been launched, the number of RISC-V users is gradually increasing, before due to some board GPU driver problems, some graphical applications can not be well played, but now, VisionFive2, MilkV Mars and other board drivers equipped with JH7110 CPU have gradually improved a lot

Godot is a powerful game engine, if the official can support and port to the RISC-V SBC, it will be a good ecological progress, and many games have the possibility of logging into RISC-V

I really hope that the official will add support for RISC-V in future releases

Thanks

**Comment by Calinou** [comment]
[Godot can already be compiled for RISC-V](https://github.com/godotengine/godot-proposals/issues/3374), but no official binaries are provided yet. [This was recently done for ARM](https://github.com/godotengine/godot-proposals/issues/988) but the RISC-V community is much smaller in comparison, and not a lot of RISC-V machines are really suited for running Godot yet (due to limited processing power or graphics drivers).
